.docs/content/1.concepts/10.plugins.md

```diff
@@ -0,0 +1,47 @@
+# Using external plugins with ArmoniK.Core
+
+## Queue adaptors as external plugins
+
+We have the possibility to have a queue adaptor as an external project and to use it as a plugin to ArmoniK.Core when needed. More about adaptors is available [here](https://github.com/aneoconsulting/ArmoniK.Core/blob/main/.docs/content/1.concepts/7.adaptors.md). Using a queue adaptor as an external plugin was possible thanks to dynamic loading of adaptors, and the use of docker images.
+
+### Loading queue adaptor into a C# project
+
+ArmoniK.Core is based on C# projects where queue adaptors are dynamically loaded. Dynamic assembly loading allows to create dotnet applications with plugins. We will refer to the application to which the plugins will be connected as the hosting application.
+
+The hosting application should define an interface that all plugins need to implement. This interface serves as a base for each plugin. In our case, the base interface that should be implemented in any plugin is in NuGet package [ArmoniK.Core.Base](https://www.nuget.org/packages/ArmoniK.Core.Base/). Any plugin should have a reference to this package.
```

Comment:
The hosting application should define an interface that all plugins need to implement and reference. This interface serves as a base for each plugin. In our case, the NuGet package  [ArmoniK.Core.Base](https://www.nuget.org/packages/ArmoniK.Core.Base/0.14.0-SNAPSHOT.194.5b51131f) should be the base interface for any plugin.

Reply:
We  "implement an interface" , "reference a package" . So plugins implement an interface that is in the package. So, they need to reference this package.